src/main/java/core/basesyntax/Balls.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Balls {
```

Comment:
classes should be named in singular not plural
```suggestion
public class Ball {
```